crypto/src/transaction/builder.rs

```diff
@@ -126,7 +126,24 @@ impl Builder {
     }
 
     /// Set the transaction fee in PEN.
-    pub fn set_fee(mut self, fee: u64) -> Self {
+    pub fn set_fee<R: RngCore + CryptoRng>(mut self, rng: &mut R, fee: u64) -> Self {
+        let pen_trace = b"PEN";
```

Comment:
Did we want to standardize on `pen` or `PEN`?

Reply:
let's use lower case in the code, and figure out a final name later